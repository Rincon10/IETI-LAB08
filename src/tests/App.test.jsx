import React from 'react';
import { shallow } from 'enzyme';
import { describe, expect, test } from '@jest/globals';
import App from '../App';

describe('Pruebas para el componente <App/>', () => {
    var wrapper = shallow(<App />);

    // before running each test
    beforeEach(() => {
        wrapper = shallow(<App />);
    });

    test('Debe de renderizar el componente', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
