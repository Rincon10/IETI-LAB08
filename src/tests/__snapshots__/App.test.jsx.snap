// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas para el componente <App/> Debe de renderizar el componente 1`] = `
<main
  id="main-container"
>
  <ContextProvider
    value={
      Object {
        "dispatchTheme": [Function],
        "theme": Object {
          "isDarkMode": false,
        },
      }
    }
  >
    <ContextProvider
      value={
        Object {
          "dispatch": [Function],
          "user": Object {
            "logged": false,
          },
        }
      }
    >
      <AppRouter />
    </ContextProvider>
  </ContextProvider>
</main>
`;
